### ISSUE TITLE
Possible to transfer the Pckage to JuliaHEP github org?

### ISSUE THREAD

**Issue by Moelf** [RobinPelkner/HS3.jl#14]
cc. @oschulz

this way we can maintain it further with JuliaHEP humanpower

**Comment by oschulz** [comment]
@RobinPelkner yes, please, could you transfer the repo? If GitHub won't let you transfer to JuliaHEP, then transfer it to me (oschulz) or Jerry (Moelf), and we'll move it.

**Comment by RobinPelkner** [comment]
I requested a transfer to @oschulz good luck and have fun

**Comment by oschulz** [comment]
Thanks @RobinPelkner , transfer complete: https://github.com/JuliaHEP/HS3.jl
